train.py

```diff
@@ -266,6 +266,8 @@
                     help='how many training processes to use (default: 1)')
 parser.add_argument('--save-images', action='store_true', default=False,
                     help='save images of input bathes every log interval for debugging')
+parser.add_argument('--distributed', action='store_true', default=False,
```

Comment:
nit: Can remove default=False if action='store_true' used

Reply:
Keeping default=False seems to adhere to the TIMM conventions. Not sure if convention is a relic or used to make the defaults more explicit to other devs.